dotnet_bridge/java/src/org/openda/dotnet/UtilsJ2NAndN2J.java

```diff
@@ -39,14 +39,14 @@ public static int OutputLevelMapJ2N(IStochModelFactory.OutputLevel outputLevel)
 			throw new RuntimeException("Unknown IStochModelFactory.OutputLevel type: " + outputLevel);
 		}
 
-		public static int ExchangeItemRoleMapJ2N(IPrevExchangeItem.Role role) {
-			if (role == IPrevExchangeItem.Role.Input) {
+		public static int ExchangeItemRoleMapJ2N(IExchangeItem.Role role) {
```

Comment:
Codacy found an issue: [The static method name 'ExchangeItemRoleMapJ2N' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)